charts/cinder-csi-plugin/values.yaml

```diff
@@ -196,5 +204,5 @@ priorityClassName: ""
 imagePullSecrets: []
 # - name: my-imagepull-secret
 
-# add annotations to all pods
+## Annotations to apply to all resources
```

Comment:
Looks like we're changing the behavior of this field here in a non backward compatible way. It would at least require a proper release note, but I'd prefer to do it without a change to how the field is interpreted.

Reply:
I added these changes because I was confused why a field with the same name does different things in the occm and cinder-csi-plugin chart. Thats why I was trying to align these annotations with the way `commonAnnotations` work in the occm chart:
See also: https://github.com/kubernetes/cloud-provider-openstack/blob/e87f50614b96a6af6095660aa6a77457223b344b/charts/openstack-cloud-controller-manager/values.yaml#L5-L6

If you would like to keep the way this field is interpreted in the cinder-csi-plugin chart the way it is right now, I would rather drop these changes from this PR instead of investing into doing this in a backward compatible way. I don't need the changes and this allows for somebody else who needs annotations on all resources to open a separate PR to solve this in a backward compatible manner.

Is this ok with you?

Reply:
After further review, I opted for changing the comment explaining what this field does instead of the behavior.